src/elasticsearch.js

```diff
@@ -0,0 +1,204 @@
+require('array.prototype.flatmap').shim();
+const { Client } = require('@elastic/elasticsearch');
```

Comment:
This can be likely replaced with `import Client from "@elastic/elasticsearch";`  but I'm not sure.